Controller/ApiController.php

```diff
@@ -40,9 +36,6 @@ class ApiController
     /** @DI\Inject("jms_translation.config_factory") */
     private $configFactory;
 
-    /** @DI\Inject */
-    private $request;
-
```

Comment:
could we create a separate PR, then rebase this on master once its merged? Those changes are no-brainers and kinda sneaking in with some fairly large unrelated changes.